app/services/clinic_finder/locator.rb

```diff
@@ -60,18 +58,6 @@ def locate_cheapest_clinic(gestational_age: 999, limit: 5)
 
     private
 
-    # Set up mutable clinic structs.
-    def build_clinic_structs
```

Comment:
I see this got demoted from a function up into a single line at 34 - can you talk me through how the one morphed into the other?

Reply:
yeah! It didn't used to be a safe assumption that we'd be loading ActiveRecord-ish objects (which respond to `attributes`) into this, so we'd have to handle the case of passing along OpenStructs. Since we've collapsed this into our rails app, it's now a pretty safe assumption, so we can simplify this handling down.